### ISSUE TITLE
More consistent archive structure

### ISSUE THREAD

**Issue by mcandre** [ziglang/zig#7351]
Could the Linux and BSD's align on whether the zig binary is nested within a `bin` directory? This is helpful for any Ansible maintainers doing scripts to install Zig, independent of the particular host involved.

**Comment by daurnimator** [comment]
> Could the Linux and BSD's align on whether the zig binary is nested within a `bin` directory?

Wouldn't zig always end up at `/usr/bin/zig`?

**Comment by andrewrk** [comment]
There are not enough details here for this issue to remain open. Slow down, open fewer, more high quality issues.

**Comment by dch** [comment]
I think can clarify what was meant here.

If you unpack [various per-platform zig tarballs](https://ziglang.org/download/), the files such as the main `zig` executable, are in different paths depending on OS & version. 

I ran into this while working with zigler, switching between v0.6.0 & v0.70.+, for example. I only found 0.7.0 FreeBSD one where it's `./bin/zig` and not just `./zig`.

A cursory check shows:

```
zig-freebsd-x86_64-0.6.0/zig
zig-freebsd-x86_64-0.7.0/bin/zig  <---------------- anomalous ---------->
zig-freebsd-x86_64-0.7.0+3599fb9bf/zig
zig-linux-aarch64-0.7.0+3599fb9bf/zig
zig-linux-x86_64-0.7.0+3599fb9bf/zig
zig-linux-x86_64-0.7.0/zig
zig-macos-x86_64-0.7.0+3599fb9bf/zig
```

I haven't comprehensively checked all the tarballs, but I think that is what the OP meant.
